crates/sui-indexer-alt-graphql/src/api/types/move_package.rs

```diff
@@ -69,16 +70,6 @@ pub(crate) struct PackageKey {
     pub(crate) at_checkpoint: Option<UInt53>,
 }
 
-/// Filter for paginating packages published within a range of checkpoints.
-#[derive(InputObject, Default, Debug)]
-pub(crate) struct CheckpointFilter {
-    /// Filter to packages that were published strictly after this checkpoint, defaults to fetching from the earliest checkpoint known to this RPC (this could be the genesis checkpoint, or some later checkpoint if data has been pruned).
-    pub(crate) after_checkpoint: Option<UInt53>,
-
-    /// Filter to packages published strictly before this checkpoint, defaults to fetching up to the latest checkpoint (inclusive).
-    pub(crate) before_checkpoint: Option<UInt53>,
-}
-
```

Comment:
Should we have different "CheckpointFilter" structs for different types that rely on checkpoint filtering? The `CheckpointFilter` here is a subset of the other. Is the goal to eventually support the same set of checkpoint filtering across the different types?